src/udiskslinuxdriveata.h

```diff
@@ -29,6 +29,16 @@ G_BEGIN_DECLS
 #define UDISKS_LINUX_DRIVE_ATA(o)    (G_TYPE_CHECK_INSTANCE_CAST ((o), UDISKS_TYPE_LINUX_DRIVE_ATA, UDisksLinuxDriveAta))
 #define UDISKS_IS_LINUX_DRIVE_ATA(o) (G_TYPE_CHECK_INSTANCE_TYPE ((o), UDISKS_TYPE_LINUX_DRIVE_ATA))
 
+/**
+ * UDISKS_LINUX_DRIVE_ATA_IS_AWAKE:
+ * @count: The power state value.
+ *
+ * Decodes the power state value as returned by #udisks_linux_drive_ata_get_pm_state.
+ *
+ * Returns: %TRUE when the drive is awake, %FALSE when sleeping.
+*/
+#define         UDISKS_LINUX_DRIVE_ATA_IS_AWAKE(count)    (count >= 0x41)
```

Comment:
`count` is a really weird name for what is actually a state.

Reply:
Well, `count` is the official name in ATA8-ACS standard. Besides `count` there are also `error`, `LBA`, `device` and `status` values returned by the `CHECK POWER MODE` ATA command. And it has been used in udisks for years...

Anyway, changed to `pm_state` where applicable.